### PR TITLE
Necromorphs will no longer speak english

### LINKED ISSUE #1336: Bug: Necros can speak english
#### Description of issue:
Necros can speak galatic common

#### Difference between expected and actual behavior:
Necros should not know common

#### Steps to reproduce:
Go to langauge tab and select galatic common as a necro

### PR DESCRIPTION
**Pull request by DTraitor**
### Purpose
Necromorphs will no longer speak english
Fixes: #1336

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/necromorph/necromorph_species.dm [hunk 1]
```diff
@@ -125,7 +125,8 @@
 	species_flags = SPECIES_FLAG_NO_PAIN | SPECIES_FLAG_NO_MINOR_CUT | SPECIES_FLAG_NO_POISON  | SPECIES_FLAG_NO_BLOCK      // Various specific features.
 	appearance_flags = 0      // Appearance/display related features.
 	spawn_flags = SPECIES_IS_RESTRICTED | SPECIES_NO_FBP_CONSTRUCTION | SPECIES_NO_FBP_CHARGEN           // Flags that specify who can spawn as this specie
-	language = LANGUAGE_NECROCHAT
+	language = null
+	default_language = "Signal Hivemind"
```
Comment (NanakoAC) [inline code/modules/mob/living/carbon/human/species/necromorph/necromorph_species.dm:129]: don't use string literals when a define exists, stick with that

why not set it in both vars too?

**Review by NanakoAC — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species/necromorph/necromorph_species.dm (hunk 1 above)
Comment (DTraitor) [reply code/modules/mob/living/carbon/human/species/necromorph/necromorph_species.dm:129]: **#define** doesnt work in this case and the game can't find language to use so it automatically sets it to english common. Var is declared in **species.dm**.

_P.S. Monkeys use the same code as I put here_

**Comment by DTraitor** [comment]
whoops, merged to wrong branch

File: code/modules/mob/living/carbon/human/species/necromorph/necromorph_species.dm (hunk 1 above)
Comment (NanakoAC) [reply code/modules/mob/living/carbon/human/species/necromorph/necromorph_species.dm:129]: well that definitely doesn't make sense. Perhaps the define is mis-spelled, try looking into it

**Comment by DTraitor** [comment]
Closing this because I couldn't find a way to use **#define** in this case (it doesnt work because it requires var in **say.dm**), **var/language** is declared in **species.dm** and changing it using simple "language = null "doesnt arranges @NanakoAC.

**Pull request closed without merging**